redaxo/src/core/lang/de_de.lang

```diff
@@ -55,6 +55,9 @@ stay_logged_in = Eingeloggt bleiben
 # redaxo/include/layout_redaxo/top.php
 logged_out  =  nicht angemeldet
 logged_in_as  =  Angemeldet als
+safemode = Safe mode
+safemode_text = Im Safe mode werden vorübergehend alle AddOns und der Autoloader deaktiviert. Falls bspw. ein AddOn nach der Installation Probleme macht, kann dieses über den Safe mode deaktiviert oder deinstalliert werden.
```

Comment:
Der Autoloader unterscheidet sich eigentlich nicht im Safe Mode.

Reply:
dann habe ich das falsch in Erinnerung gehabt, als das mal erklärt wurde.